contracts/game_token/lib.rs

```diff
@@ -14,19 +14,22 @@ pub mod game_token {
         codegen::{EmitEvent, Env},
         reflect::ContractEventBase,
     };
-    use ink_prelude::{format, string::String};
+    use ink_prelude::{format, string::String as Strink};
     use ink_storage::traits::SpreadAllocate;
     use openbrush::{
-        contracts::psp22::{extensions::metadata::*, Internal},
-        traits::Storage,
+        contracts::psp22::{
+            extensions::{burnable::*, metadata::*, mintable::*},
+            Internal,
+        },
+        traits::{Storage, String},
```

Comment:
a wot szto, traits::String?

Reply:
it's alias for `Vec<u8>` xD

Reply:
Conveniently defined in `traits` module, just to, you know, be easier to follow :P